tests/jiva/jiva.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The OpenEBS Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import "flag"
+
+var (
+	// KubeConfigPath is the path to
+	// the kubeconfig provided at runtime
+	KubeConfigPath string
+	// JivaReplicaCount is the value of
+	// replica count provided at runtime
+	JivaReplicaCount int
```

Comment:
ReplicaCount is enough since Jiva is mentioned in the folder already.

Reply:
Done